### PR TITLE
Link cellrank in docstrings

### LINKED ISSUE #269: Add links to cellrank
Hi @VolkerBergen, would be great to place links to cellrank.org in the scvelo docs for the following functions:
-  `scv.tl.terminal_states`
- `scv.pl.heatmap`. CellRank has a heatmap function [here](https://cellrank.readthedocs.io/en/latest/api/pl/cellrank.pl.heatmap.html#cellrank.pl.heatmap) that allows you to plot gene expression trends specific to lineages, sorted in latent time, just like in scvelo. 
- `scv.utils.get_extrapolated_state`, `utils.get_cell_transitions` --> potentially good link to absorption probabilities. 

I would suggest to just link to the index page of cellrank.org.

### PR DESCRIPTION
**Pull request by michalk8**
Hi @VolkerBergen ,

I believe you've discussed this with Marius a while back, sorry it took me so long.
@Marius1311 - is tie phrasing fine? I've tried to be very minimal.

closes #269

### CODE REVIEW

File: scvelo/tools/terminal_states.py [hunk 1]
```diff
@@ -227,10 +227,14 @@ def terminal_states(
...
         scv.tl.terminal_states(adata)
-        scv.pl.scatter(adata, color=[ 'root_cells', 'end_points'])
+        scv.pl.scatter(adata, color=['root_cells', 'end_points'])
 
     .. image:: https://user-images.githubusercontent.com/31883718/69496183-bcfdf300-0ecf-11ea-9aae-685300a0b1ba.png
 
+    Other option would be to use :func:`cellrank.tl.terminal_states` (end points)
+    or :func:`cellrank.tl.initial_states` (root points), which uses the real Schur decomposition
+    instead of the eigendecomposition because of the asymmetricity of the transition matrix.
+
```
Comment (Marius1311) [inline scvelo/tools/terminal_states.py:237]: Better: "An alternative option would be to use XXX or XXX, both of which make use of real Schur decompositions to handle non-symmetries in the transition matrix. These methods are also able to separate multiple initial/terminal states and to pick the most likely cells from each. "

File: scvelo/tools/transition_matrix.py [hunk 2]
```diff
@@ -36,6 +36,9 @@ def transition_matrix(
     from the velocity graph :math:`\\pi_{ij}`, with row-normalization :math:`z_i` and
     kernel width :math:`\\sigma` (scale parameter :math:`\\lambda = \\sigma^{-1}`).
 
+    Other option would be to use :func:`cellrank.tl.transition_matrix`
+    which can also take into account velocity uncertainty.
+
```
Comment (Marius1311) [inline scvelo/tools/transition_matrix.py:41]: Again, "An alternative option is given by XXX which takes into account uncertainty in the velocity vectors when computing transition probabilities.  "

**Review by Marius1311 — CHANGES_REQUESTED** [review]

**Comment by michalk8** [comment]
Changed it as you've suggested - should be good now.

**Review by Marius1311 — APPROVED** [review]

**Comment by Marius1311** [comment]
Looking good from my point of view. @VolkerBergen , could you take a look at this please?

**Comment by michalk8** [comment]
Ping @VolkerBergen

**Comment by Marius1311** [comment]
Would be great to merge this, it's only some very minor changes that only affect the docs. @VolkerBergen

**Comment by VolkerBergen** [comment]
thank you, slightly modified and pushed.

**Comment by Marius1311** [comment]
thanks Volker.

**Pull request closed without merging**